tests/network/sriov.go

```diff
@@ -461,7 +431,15 @@ var _ = Describe("[Serial]SRIOV", func() {
 			Expect(err).ToNot(HaveOccurred())
 
 			//create two vms on the same sriov network
-			vmi1, vmi2 := createSriovVMs(sriovnetLinkEnabled, sriovnetLinkEnabled, cidrA, cidrB)
+			vmi1, err := createSRIOVVmiOnNode(sriovNode, sriovnetLinkEnabled, cidrA)
+			Expect(err).ToNot(HaveOccurred())
+			vmi2, err := createSRIOVVmiOnNode(sriovNode, sriovnetLinkEnabled, cidrB)
+			Expect(err).ToNot(HaveOccurred())
+
+			vmi1, err = waitVMI(vmi1)
```

Comment:
why not do `createSRIOVVmiOnNodeAndWait` like done in previous commit?

Reply:
The original `createSriovVMs` has created two VM/s and only then waited from them.
This is doing the same, starting the VM/s in parallel and waiting for both afterwards.

Reply:
just saying it would reduce caller code :) ..

Reply:
Reducing caller code vs reducing test time. Usually the later wins, don't you agree?